difference_of_squares.rb

```diff
@@ -0,0 +1,29 @@
+class Squares
+  def initialize(nutual_num)
+    @nutual_num=nutual_num
```

Comment:
@keyurgohil2426 Space issue `=` please follow it for each and every program you write.